### DISCUSSION TITLE
Deploy multiple repos to one project

### DISCUSSION THREAD

**Discussion by awolfe76** [vercel/community#147] [Help] [answered]
Currently in a project you can define one repo to deploy, this makes perfect sense.

But, with a recent Next.js version you can now set a [`basePath`](https://nextjs.org/docs/api-reference/next.config.js/basepath) to be able to have an application served from a subdirectory.

What I'm interested in doing is having multiple applications deployed to the same domain (or Vercel project) with one application being the "main" application and any other application requiring the `basePath` config to be set.

My scenario is just that...

I have a Next.js application served from `mydomain.com`. I have another Next.js application (separate repository, because it's also a fairly large application) that I'd ideally like to serve from `mydomain.com/the-other-app`. Currently I have to deploy `the-other-app` as a different project, different domain.

Maybe it would trigger a full re-build of both applications? Possible to re-build and deploy only one at a time, whichever had a change?

Just curious to know if this is possible.

_Both Next.js and Vercel are 💯  ⭐ , so thank you!_

**Comment by sanozie** [comment, marked as answer]
I have a very similar case, I've been looking for a way to configure ```mydomain.com``` with one repository and ```app.mydomain.com``` with another, and have it all managed in the same project. For now I'll probably set it up as seperate projects

**Reply by sanozie** [reply]
Guess I'll leave an update of what I did:
Since app.domain.com was technically a new domain, I went with making a separate project. Until this is a feature, creating sub domains and making new projects might be the only way to do this.

**Reply by Pieparker** [reply]
Thanks for sharing. If it was the _same_ repository but a different branch, that's very easily doable from within the domains UI.
